lib/transport_web/templates/dataset/details.html.eex

```diff
@@ -11,7 +11,7 @@
           <div class="shortlist__link shortlist__link--external-link">
             <i class="icon icon--link" aria-hidden="true"></i>
             <%= link(dgettext("page-shortlist", "See on data.gouv.fr"),
-                     to: "https://www.data.gouv.fr/datasets/#{@dataset.slug}/",
+                     to: "https://www.data.gouv.fr/datasets/#{@dataset.id}/",
```

Comment:
Same, this is our front's representation, it not the same context than our internal relationship with data.gouv.fr.